doc/source/whatsnew/v1.0.2.rst

```diff
@@ -28,6 +28,11 @@ Fixed regressions
 Bug fixes
 ~~~~~~~~~
 
+**ExtensionArray**
+
+- Fixed issue where taking the minimum or maximum of a ``StringArray`` or ``Series`` with ``StringDtype`` type would raise. (:issue:`31746`)
```

Comment:
does this also close #18588 can you add a note for that

Reply:
Yes, it does close that issue